src/test/java/com/softserveinc/ita/HomePageTest.java

```diff
@@ -19,4 +21,20 @@ public void verifyLanguageSwitchingTest() {
                 .as("Language should be switched")
                 .isTrue();
     }
+
+    @Test
+    public void verifySearchTest() {
+        var requiredItem = "Xiaomi Redmi Note 10";
+        homePage.searchBarInputField(requiredItem);
+        homePage.clickSearchButton();
+        var firstItem = homePage.getFirstRequiredItem(requiredItem);
+        var lastItem = homePage.getLastRequiredItem(requiredItem);
+        AssertionsForClassTypes.assertThat(firstItem)
```

Comment:
add empty line before assertion, use static import

Reply:
old version of commit, it was already fixed